infrastructure/tooling/src/build/tasks/monoimage.js

```diff
@@ -166,7 +166,7 @@ const generateMonoimageTasks = ({ tasks, baseDir, cmdOptions, credentials, logsD
       await writeRepoFile('temp/devel-image/Dockerfile', [
         `FROM ${requirements['monoimage-docker-image']}`,
         'RUN npm install --global nodemon',
-        'RUN yarn install && yarn cache clean --all',
+        'RUN corepack enable && yarn set version berry && yarn install && yarn cache clean --all',
```

Comment:
who is `corepack`?

Reply:
It's a really neat new feature in nodejs that helps manage the package managers.

Two key core workflows it simplifies (from https://nodejs.org/api/corepack.html):

  * It eases new contributor onboarding, since they won't have to follow system-specific installation processes anymore just to have the package manager you want them to.
  * It allows you to ensure that everyone in your team will use exactly the package manager version you intend them to, without them having to manually synchronize it each time you need to make an update.